### PR TITLE
Delete 'pricing faqs' doc, and improve general pricing doc

### PR DESCRIPTION
**Pull request by zuluecho9**
Work done: 

* Delete pricing FAQ license doc, and move out some info about free/unpaid to the license doc section on that
* General improvement to the pricing doc, including: adding section on add-on pricing, and on editions
* Add section to main pricing doc about understanding the order form terms, which was moved from the FAQ doc.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 1]
```diff
@@ -12,12 +12,14 @@ redirects:
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-one-pricing-billing
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing
   - /docs/accounts/accounts-billing/new-relic-one-pricing-billing
+  - /docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions
+  - /docs/licenses/license-information/faq/frequently-asked-questions
 ---
 
-New Relic bills on just two factors: ingested data and billable users. In this doc, we'll explain how our pricing model works, what you get for free, and other details. 
+New Relic has two primary billing factors: ingested data and billable users. In this doc, we'll explain how pricing works, what you get for free, and other details.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:19]: @zuluecho9 This implies that there are some secondary billing factors...

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 2]
```diff
@@ -26,16 +28,27 @@ If you're not yet a New Relic customer and are just getting started learning abo
 
 ## How our pricing works [#how-pricing-works]
 
-Our usage-based pricing model is simple, which allows you to accurately predict your billing. It bills on these two factors:
+Our usage-based pricing model is simple, which allows you to accurately predict your billing. It bills on these two primary factors:
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:31]: @zuluecho9 I'm not sure "primary" adds much here. I'd just leave it the way it was.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 3]
```diff
@@ -26,16 +28,27 @@ If you're not yet a New Relic customer and are just getting started learning abo
...
 * Your [billable user count](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing). Core users and full platform users are billable. Basic users are free. Your per-user cost depends on your pricing edition. Free tier and Standard edition get one free full platform user.
 
-Besides determining the billable user cost, our pricing editions (Standard, Pro, Enterprise) give you access to some different features and limits (for example, different limits for synthetic monitor checks, and access to FedRAMP- and HIPAA- compliant data). For more details, see the [pricing page](https://newrelic.com/pricing). 
+### Pricing editions [#editions]
+
+Besides determining the cost of your billable users, the editions (Standard, Pro, Enterprise) gates access to different features and limits. For example, the editions have different limits for synthetic monitor checks. Also, Pro and Enterprise edition have several features not available on Standard, like FedRAMP- and HIPAA- compliant data.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:38]: @zuluecho9 shouldn't it be "gate access"?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 4]
```diff
@@ -12,12 +12,14 @@ redirects:
...
+  - /docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions
+  - /docs/licenses/license-information/faq/frequently-asked-questions
 ---
 
-New Relic bills on just two factors: ingested data and billable users. In this doc, we'll explain how our pricing model works, what you get for free, and other details. 
+New Relic has two primary billing factors: ingested data and billable users. In this doc, we'll explain how pricing works, what you get for free, and other details. 
 
 <Callout variant="tip">
-This doc is about our newer usage-based pricing model, and not our original pricing model. Not sure which model you're on? See [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
+This doc is about our newer usage-based pricing model, and not our original pricing model. You're probably on our newer pricing model, but if you're not sure, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
```
[the remark this answers is not in the document]
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:22]: unfortunately it's a bit more complicated. some of the older organizations have switched to the new one. and for large organizations, they may have created sub-accounts for various departments on the old pricing model. for a few reasons it's hard to quickly describe it. i'm basing this approach on something I once saw on Apple, to try to quickly summarize the situation.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:19]: unfortunately there are secondary billing factors (various add-on things). so there are two primary, and several secondary

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 5]
```diff
@@ -26,16 +28,27 @@ If you're not yet a New Relic customer and are just getting started learning abo
...
+Besides determining the cost of your billable users, the editions (Standard, Pro, Enterprise) gates access to different features and limits. For example, the editions have different limits for synthetic monitor checks. Also, Pro and Enterprise edition have several features not available on Standard, like FedRAMP- and HIPAA- compliant data. 
+
+For more details on editions, see the [pricing page](https://newrelic.com/pricing). 
+
+### Add-ons [#add-ons]
+
+In addition to the primary billing factors mentioned above, there are several optional add-ons that are billable. These include: 
+
+* Exceeding the [default number of synthetic monitor checks](/docs/synthetics/synthetic-monitoring/getting-started/monitor-limits)
+* Exceeding the [default data retention](/docs/data-apis/manage-data/manage-data-retention/#adjust-retention)
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:47]: I feel like there's a word missing after "retention"... "default data retention time"?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 6]
```diff
@@ -63,25 +76,51 @@ For how to find billing and usage information in the UI, see [Billing-related UI
...
-## User count calculation [#user-count]
+## How billable users are calculated [#user-count]
 
-To learn how billable users are calculated, see [User count calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing).
+See [Billable user calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing).
 
 ## Billing schedule options [#usage-plans]
 
-A New Relic organization can have one of several types of plans that govern how billing scheduling is handled. Here are the main options:
+A New Relic organization can have one of several types of plans that govern how your billing is scheduled and implemented. Here are the main options:
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:91]: @zuluecho9 are there non-main options?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 7]
```diff
@@ -63,25 +76,51 @@ For how to find billing and usage information in the UI, see [Billing-related UI
...
 
 * **Pay-as-you-go**: With this option, billing happens at the end of each calendar month. There are no payment commitments past the current month and an organization can cancel at any time. An organization's first end-of-month bill includes billable usage starting from when their credit card information was input into New Relic.
 * **Commitment contracts**: Commitment contract options include our Volume Plan, Savings Plan, and other options. With commitment contracts, an estimate of long-term expected usage is created and there's an agreed upon payment commitment. If an organization's usage exceeds that commitment, the excess usage is billed at the previously agreed-upon rates. These are available primarily to Pro and Enterprise edition organizations. 
 
 For more about how these options can affect your user downgrade rules, see [Downgrade rules](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#user-downgrade-rules).
 
+## Understand order terms [#order]
+
+Your order and invoice contain several details about your pricing plan. Here's an example of an item found on a New Relic order:
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:100]: @zuluecho9 are order and invoice two separate things or one thing?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 8]
```diff
@@ -63,25 +76,51 @@ For how to find billing and usage information in the UI, see [Billing-related UI
...
+
+Your order and invoice contain several details about your pricing plan. Here's an example of an item found on a New Relic order: 
+
+```
+New Relic Volume Plan - Pro Full Platform Users
+```
+
+Here's a breakdown of these terms: 
+
+* The `New Relic` means you're on our [primary user billing version](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing/#pricing-versions). (If this is `New Relic One`, that means you're on our original user billing version.)
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:108]: @zuluecho9  the "The" at the start of these bullets seems odd and I think probably you can delete them and just start with the term itself.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 9]
```diff
@@ -96,9 +135,11 @@ Note that the billable users in lower tiers remain billable at that tier's cost,
 
 If you want access to tiered pricing, talk to your account representative. 
 
-## Opting in to the core user release [#core-acceptance]
+## Core user release [#core-acceptance]
+
+In January of 2022, we released a third [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type): the core user. New Relic organizations created after the core user release had access to that user type and the associated [changes to billing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing/#pricing-versions). Existing organizations had the option to enable core users by opting in via an acceptance form called the `core amendment`. For some organizations, this acceptance process is available in the UI, on the **Manage your plan** UI page.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:140]: @zuluecho9 How do I know if I'm one of the organizations that can access the acceptance process in the UI?

File: src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx [hunk 10]
```diff
@@ -472,7 +471,7 @@ Details:
     * Pro
     * Enterprise
 
-      A New Relic organization can only be on one edition at a time. Your Order will indicate your edition.
+    A New Relic organization can only be on one edition at a time. Your Order will indicate your edition.
```
Comment (paperclypse) [inline src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx:474]: @zuluecho9 Feels odd to capitalize "Order"

**Review by paperclypse — APPROVED** [review]
@zuluecho9 Left some feedback on this. Definitely check out those Vale linter comments and fix where you can.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx (hunk 6 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:91]: yeah, unfortunately

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx (hunk 9 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:140]: it's pretty complicated, and it's best not to get into that.

File: src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx (hunk 10 above)
Comment (zuluecho9) [reply src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx:474]: those are the license docs, and they do all sorts of legal-speak things like that

**Pull request merged**